src/route.c

```diff
@@ -717,6 +687,16 @@ static int route_del_msg_cb(struct dpvs_msg *msg)
     return route_msg_process(false, msg);
 }
 
+static struct dpvs_sockopts agent_route_sockopts = {
+    .version        = SOCKOPT_VERSION,
+    .set_opt_min    = DPVSAGENT_ROUTE_ADD,
+    .set_opt_max    = DPVSAGENT_ROUTE_DEL,
+    .set            = route_sockopt_set,
+    .get_opt_min    = DPVSAGENT_ROUTE_GET,
+    .get_opt_max    = DPVSAGENT_ROUTE_GET,
+    .get            = route_sockopt_get,
+};
+
```

Comment:
The action functions of `agent_route_sockopts ` are the same with `route_sockopts`. Can they be combined?

Reply:
the definition of sockopt about dpvs agent is a isolated area.  it looks more clearly